regulations/tests/views_preamble_tests.py

```diff
@@ -208,3 +212,35 @@ def test_add_amendment(self, fetch_meta, fetch_toc):
                 title='99', part='222', name='Some title for reg 222',
                 authority_url='/preamble/docdoc/cfr_changes/222',
                 sections=[])])
+
+    def test_navigation(self):
+        toc = [
+            preamble.ToCPart(
+                part='478',
+                title='27',
+                name='Commerce',
+                authority_url='',
+                sections=[
+                    preamble.ToCSect(
+                        full_id='2016_02749-cfr-478-99',
+                        url='/preamble/2016_02749/cfr_changes/478-99',
+                        title=u'§ 478.99 Certain prohibited',
+                        section='99',
+                    ),
+                    preamble.ToCSect(
+                        full_id='2016_02749-cfr-478-120',
+                        url='/preamble/2016_02749/cfr_changes/478-120',
+                        title=u'§ 478.120 Firearms',
+                        section='120',
+                    ),
+                ],
+            )
+        ]
+
+        nav = preamble.cfr_section_navigation('478-99', toc)
+        assert_equal(nav['next'].section_id, '2016_02749-cfr-478-120')
+        assert_is_none(nav['previous'])
```

Comment:
478.99 wouldn't ever be the "first" section; we'll always have an "Authority":
<img width="249" alt="screen shot 2016-04-26 at 2 55 46 pm" src="https://cloud.githubusercontent.com/assets/326918/14830494/fdfdaca0-0bbe-11e6-844c-8cb1b7b33f20.png">